knplabs/dictionary-bundle/2.0/config/packages/knp_dictionary.yaml

```diff
@@ -0,0 +1,2 @@
+knp_dictionary:
+    dictionaries: []
```

Comment:
Is it likely that a user will add more dictionaries here?

What would happen if we did not use this config?

Reply:
.... nothing ....